app/app.go

```diff
@@ -404,7 +405,9 @@ func New(
 
 	if manager := app.SnapshotManager(); manager != nil {
 		err = manager.RegisterExtensions(
-			wasmkeeper.NewWasmSnapshotter(app.CommitMultiStore(), &app.AppKeepers.WasmKeeper),
+			// TODO: is this okay? (may not matter since we are sharing the VM instance)
+			// wasmkeeper.NewWasmSnapshotter(app.CommitMultiStore(), &app.AppKeepers.WasmKeeper),
+			wasmlckeeper.NewWasmSnapshotter(app.CommitMultiStore(), &app.AppKeepers.WasmClientKeeper),
```

Comment:
Waiting on a resp from the IBC team about which to use. I would assume the x/wasm keeper